### ISSUE TITLE
sorting - no order changed when it should

### ISSUE THREAD

**Issue by mmwo** [Urigo/meteor-angular2.0-socially#164]
Hi, followed the tutorial to implement sort for subscribed collection.
Anyway the sort is not working as expected.
I tried the code from zip in **step 13** and there is the same issue

Lets sey i have 3 parties

```
1. All have fun
2. Hey! Hi! Hello!
3. Public zone 

```

When I change the name of the party the view is updated with the new name but the order is as it was before.

I see in the view:

```
1. Zone <-- i expected this party to be at the end of the list
2. Hey! Hi! Hello!
3. Public zone
```

 Party with changed name is not reordered on the list. Nothing helps to change it, jumping between routes, even page refresh. As if the  name change was never done. 

When I trigger sorting by myself i get the list like below:

```
1. Public zone 
2. Hey! Hi! Hello!
3. Zone <-- i expected this party to be on the top as it starts with 'Z' character
```

There is no error in console.

I would like to have my list to be refreshed when there is change of the name.
Can you help me solve it?

**Comment by dotansimha** [comment]
@Jitra you are right.
The data need to trigger the sort change, verified it with a clean Meteor application. 
I will update soon.

**Comment by dotansimha** [comment]
@Jitra It's a bug in `meteor-rxjs`, and I fixes it in: https://github.com/Urigo/meteor-rxjs/pull/21
We will soon merge, publish and update the tutorial :) 

Keeping open until completely solved.

**Comment by dotansimha** [comment]
@Jitra 
We fixed the bug in `meteor-rxjs` and released version 0.4.0 of it.
I also updated the tutorial with the latest version and now it works great!

Thanks again for spotting the bug!
